docs/changelog.md

```diff
@@ -4,8 +4,84 @@ title: Changelog
 sidebar_label: Changelog
 ---
 
+## v2.4.0
+
+### Major Features and Improvements
+
+- Add replica state to `SHOW REPLICAS` query.
+  [#379](https://github.com/memgraph/memgraph/pull/379)
+- Add `current_timestamp` and `number_of_timestamp_behind_master` to `SHOW
+  REPLICAS` query. [#412](https://github.com/memgraph/memgraph/pull/412)
+- Add frequent replica ping. `main` instance checks state of the replicas with
+  given frequency controller by `--replication-replica-check-delay-sec`. The
+  check allows latest information about the state of each replica from `main`
+  point of view. [#380](https://github.com/memgraph/memgraph/pull/380)
```

Comment:
This is also 2.3.1, since https://github.com/memgraph/memgraph/pull/380 has a commit 8e3ab1ad0f1996392ff8218255a16a9ddc2d097a and that shows in release/2.3 branch https://github.com/memgraph/memgraph/commits/release/2.3